lib/routes/routesUtils.js

```diff
@@ -59,9 +59,14 @@ function okHeaderResponse(headers, response, httpCode, log) {
  * @return {object} response - response object with additional headers
  */
 function okXMLResponse(xml, response, log, additionalHeaders) {
+    const bytesSent = Buffer.byteLength(xml);
     log.trace('sending success xml response');
+    log.addDefaultFields({
+        bytesSent,
+    });
     setCommonResponseHeaders(additionalHeaders, response, log);
-    response.writeHead(200, { 'Content-type': 'application/xml' });
+    response.writeHead(200, { 'Content-type': 'application/xml',
+        'Content-length': bytesSent });
```

Comment:
let's actually not add the content length header since aws returns transfer encoding (that's why your test was failing)